mne/inverse_sparse/mxne_optim.py

```diff
@@ -768,6 +768,133 @@ def norm_l1_tf(Z, shape, n_orient):
     return l1_norm
 
 
+def norm_epsilon(Y, l1_ratio, n_steps):
+    """Dual norm of (1. - l1_ratio) * L2 norm + l1_ratio * L1 norm, at Y.
```

Comment:
It is not exactly evaluated at Y: entries of Y which corresponds to frequencies different from the 1st and last are counted twice.